ci/autoresearch/phases.py

```
@@ -1525,16 +1532,24 @@ async def _run_bring_up_tests(ctx: RunContext) -> int:
                 else f"   ❌ JSON-RPC echo — result mismatch"
             )
             print(
-                f"   ✅ FastLED log pipeline — FL_DBG emitted via same Serial transport"
+                f"   ✅ FL_WARN_LIT — FastLED warn pipeline reached host"
```

Ruff (0.15.15)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)